### DISCUSSION TITLE
1980-05-29-totp/

### DISCUSSION THREAD

**Discussion by giscus** [FanzOfLenaZavaroni/fanzoflenazavaroni.github.io#45] [General]
# 1980-05-29-totp/

David ‘Kid’ Jensen presents chart hits of the week. Guests include Liquid Gold, Hot Chocolate, Elton John, Don McLean, Thin Lizzy, Roxy Music, OMD, Jermaine Jackson, Stiff Little Fingers, Mystic Merlin and Lena Zavaroni. Includes a dance sequence by Legs & Co.

https://fanzoflenazavaroni.github.io/1980-05-29-totp/

**Comment by StephenMolloy1** [comment]
I's a shame that they can't  indicate in the programme summary that they are airing a shortened version without Lena and some other artistes.

**Reply by FanzOfLenaZavaroni** [reply]
Sorry for the delay in replying.

There is no reason why they could not add an indicator to inform visitors to their website that the episodes being broadcast are the shortened versions of TOTP but that would take a level of caring that the BBC does not have.

An indicator of some sort must be active within the code that powers their website for iPlayer to know which version to provide to viewers so that same indicator could also add " (Shortened Version)" to the end of the programme title that would be better than nothing but I think a separate page would be better after all they have a separate page for TOTP2 and as all the current broadcasts of TOTP are all  shortened we the paying public should know what is actually being shown.

Their laziness makes it very hard for anyone wishing to make an archive of broadcast content that is accurate.

TOTP, [...Sings Neil Diamond](https://fanzoflenazavaroni.github.io/2010-11-14-sings-neil-diamond/#infobox3), [The Morecambe And Wise Show](https://fanzoflenazavaroni.github.io/1976-02-11-the-morecambe-and-wise-show/) all have misleading information along with many other programmes.
